gitops-deployment/wsa/templates/wsa-health.yaml

```
@@ -0,0 +1,12 @@
+{{- if .Values.wsaHealth.spec.license.accept}}
+apiVersion: automation.websphere.ibm.com/v1
+kind: WebSphereHealth
+metadata:
+  name: wsa-health
+  namespace: {{.Values.subscription.wsaInstanceNamespace}}
+  annotations:
+    argocd.argoproj.io/sync-wave: "50"
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+spec:
+  {{.Values.wsaHealth.spec | toYaml | indent 2| trim}}
```

Very minor style nit

```suggestion
  {{.Values.wsaHealth.spec | toYaml | indent 2 | trim}}
```

---

And also in the same places below.